src/platform/backends/qemu/qemu_platform.h

```diff
@@ -54,11 +54,11 @@ class QemuPlatform : private DisabledCopyMove
     {
         return {};
     };
-    virtual std::vector<NetworkInterfaceInfo> networks() const
+    virtual std::vector<NetworkInterfaceInfo> networks() const = 0;
+    virtual std::string create_bridge_with(const std::string& interface) const
     {
-        throw NotImplementedOnThisBackendException("networks");
+        return interface;
```

Comment:
This just echoes the string because the bridge gets handled by the qemu bridge helper, right?

Reply:
No, in fact, the simple implementation is not used on Linux. I've done the function pure virtual.